ansible/roles/miniflux/defaults/main.yml

```diff
@@ -0,0 +1,307 @@
+    group: '{{ miniflux__group }}'
+    home: '{{ miniflux__home }}'
+    system: True
+
+                                                                   # ]]]
+# .. envvar:: miniflux__postgresql__dependent_extensions [[[
+#
+# Configuration of PostgreSQL extensions for the
+# :ref:`debops.postgresql` Ansible role.
+miniflux__postgresql__dependent_extensions:
+
+  - database: '{{ miniflux__database_name }}'
+    extension: 'hstore'
+
+                                                                   # ]]]
+# .. envvar:: miniflux__golang__dependent_packages [[[
+#
+# Configuration for the :ref:`debops.golang` Ansible role.
+miniflux__golang__dependent_packages:
+
+  - name: 'miniflux'
+    upstream_type: '{{ miniflux__upstream_type }}'
+    url:
+      - src: '{{ miniflux__upstream_url_mirror + "download/2.0.35/miniflux-" + miniflux__upstream_platform }}'
+        dest: 'releases/{{ miniflux__upstream_platform }}/miniflux/miniflux/2.0.35/miniflux_2.0.35_linux_amd64'
+    url_binaries:
+      - src: 'releases/{{ miniflux__upstream_platform }}/miniflux/miniflux/2.0.35/miniflux_2.0.35_linux_amd64'
+        dest: 'miniflux'
+        notify: [ 'Restart miniflux' ]
+    git:
+      - repo: '{{ miniflux__upstream_git_repository }}'
+        version: '2.0.35'
+        build_script: |
+          make clean linux-amd64
+    git_binaries:
+      - src: 'github.com/miniflux/v2/bin/miniflux.app'
+        dest: 'miniflux'
+        notify: [ 'Restart miniflux' ]
+
+                                                                   # ]]]
+# .. envvar:: miniflux__nginx__dependent_upstreams [[[
+#
+# Upstream configuration for the :ref:`debops.nginx` Ansible role.
+miniflux__nginx__dependent_upstreams:
+
+  - name: 'miniflux-upstream'
+    server: '{{ miniflux__bind }}:{{ miniflux__port }}'
+
+                                                                   # ]]]
+# .. envvar:: miniflux__nginx__dependent_servers [[[
+#
+# Server configuration for the :ref:`debops.nginx` Ansible role.
+miniflux__nginx__dependent_servers:
+
+  - name: '{{ miniflux__fqdn }}'
+    by_role: 'debops.miniflux'
+    filename: 'debops.miniflux'
+    type: 'proxy'
+    proxy_location: '/'
+    proxy_headers: True
+    proxy_options: |
+      proxy_redirect off;
+    proxy_pass: 'http://miniflux-upstream'
+
+                                                                   # ]]]
+# .. envvar:: miniflux__etc_services__dependent_list [[[
+#
+# Configuration for the :ref:`debops.etc_services` Ansible role.
+miniflux__etc_services__dependent_list:
+
+  - name: 'miniflux'
+    port: '{{ miniflux__port }}'
+
+                                                                   # ]]]
+                                                                   # ]]]
+                                                                   # ]]]
+
+# TODO: Create documentation for these used variables
+#
+miniflux__comment: 'Miniflux'
+miniflux__bind: '127.0.0.1'
+miniflux__port: '3366'
```

Comment:
Looking forward to it. :-)

Reply:
Default Miniflux configuration uses 127.0.0.1:8080. Do I really need to set some variables for `__bind` and `__port`? I was thinking maybe I can use UNIX socket for nginx and Apache?

Reply:
You can use a socket, but it would be nice to have it configurable so that if somebody want to do a distributed setup, they still can. If one variable is enough to define that socket/port connection then you can go with that; otherwise if Miniflux uses bind address and port separately, then can be separate.